src/ui/qgsvectortileconnectiondialog.ui

```diff
@@ -67,43 +67,71 @@
         </property>
        </widget>
       </item>
-      <item row="1" column="2">
-       <widget class="QLineEdit" name="mEditUrl">
-        <property name="toolTip">
-         <string>URL of the connection, {x}, {y}, and {z} will be replaced with actual values. Use {-y} for inverted y axis.</string>
+      <item row="2" column="0" colspan="2">
+       <widget class="QCheckBox" name="mCheckBoxZMin">
+        <property name="text">
+         <string>Min. Zoom Level</string>
         </property>
-        <property name="placeholderText">
-         <string>http://example.com/{z}/{x}/{y}.pbf</string>
+        <property name="checked">
+         <bool>true</bool>
         </property>
        </widget>
       </item>
-      <item row="6" column="0" colspan="2">
-       <widget class="QCheckBox" name="mCheckBoxZMax">
+      <item row="7" column="0" colspan="3">
+       <widget class="QGroupBox" name="mAuthGroupBox">
```

Comment:
Glad to see you've added authentication to the 'generic' vector tile dialog too, thanks for that.